src/pint/utils.py

```diff
@@ -3026,7 +3026,7 @@ def woodbury_dot(
 
     logdet_N = np.sum(np.log(Ndiag))
     logdet_Phi = np.sum(np.log(Phidiag))
-    _, logdet_Sigma = np.linalg.slogdet(Sigma)
+    _, logdet_Sigma = np.linalg.slogdet(Sigma.astype(float))
```

Comment:
Are there no cases where long double precision is needed for this?

Reply:
I don't think so. For comparison, ENTERPRISE uses only double precision for the same computation.

Reply:
More importantly, the `np.longdouble` type is unsupported in this function.